README.md

```diff
@@ -39,69 +43,131 @@ Pre-built docker for popular AI frameworks. Easy to include heterogeneous hardwa
 
-- The OpenPAI user [documentation](./docs/documentation.md) provides in-depth instructions for using OpenPAI
-- Visit the [release notes](https://github.com/Microsoft/pai/releases) to read about the new features, or download the release today.
-- [FAQ](./docs/faq.md)
+* Choose training environment. OpenPAI uses [Docker](https://www.docker.com/) to provide runtime environment.
+
+    You can refer to [here](https://hub.docker.com/r/ufoym/deepo) to find more deep learning environments, for example, ufoym/deepo:pytorch-py36-cu90. Note, it doesn't include openssh-server, curl. So you want to use ssh with those docker images, you need use `apt install openssh-server curl` in command field to enable ssh.
+
+* Put code and data in. OpenPAI creates a clean environment, you need to use co mmand field to copy data and code in, and start training. You can join multiple commands with `&&`. If extra system or Python components are needed, they can be installed in the command by `apt install` or `python -m pip install`.
+
+    There are some suggestions to exchange data with running environment, but you should check with OpenPAI administrator, which kind of storage is supported, and recommended approach to access it. Refer to [here](TODO provide storage solution) for sample solutions.
+
+* Copy model out. It's similiar with above topic, if code and data can copy in, model can also be copied out.
+
+* Running distributed training job. OpenPAI can allocate multiple environments to same job to support distributed training.
+
+If you want to learn more, refer to [here](docs/job_tutorial.md#write-a-job-json-configuration-file-) for more information on job definition.
+
+### OpenPAI client
+
+Rather than web portal, and RESTful API, OpenPAI have a friendly client tool for user. It's an extension of [Visual Studio Code](https://code.visualstudio.com/), called `OpenPAI VS Code Client`. Once you installed Visual Studio Code, you can search and install it in extensions tab.
+
+`OpenPAI VS Code Client` can submit job, simulate job running locally, manage multiple OpenPAI environments, and so on. Refer to [here](https://marketplace.visualstudio.com/items?itemName=OpenPAIVSCodeClient.pai-vscode) for more information.
+
+### Troubleshooting job failure
+
+You can use Web portal and job log to analyze job failure, also you can log into environment with SHH to do it.
+
+Refer to [here](docs/job_tutorial.md#how-to-debug-the-job-) for more informaiton of troubleshooting job failure on OpenPAI. It's recommended to get job running successfully locally, and then submit it to OpenPAI. So that it doesn't need to troubleshoot code problems remotely.
+
+## Administration
+
+* [Manage cluster with paictl](docs/paictl/paictl-manual.md)
+* [Monitoring](./docs/webportal/README.md)
+
+## Reference
+
+* [Job definition](docs/job_tutorial.md#write-a-job-json-configuration-file-)
+* [RESTful API](docs/rest-server/API.md)
+* Design documents could be found [here](docs).
+
```

Comment:
that link are not only for "Design" docs...

Reply:
Yes, it just a temporary link to design docs. So that developer knows where to find them. It will be more specific in next updates.